padertorch/contrib/examples/tasnet/tasnet.py

```diff
@@ -226,3 +226,54 @@ def review(self, inputs, outputs):
 
     def flatten_parameters(self) -> None:
         self.dprnn.flatten_parameters()
+
+
+class ModularTasNet(TasNet):
+    def __init__(
```

Comment:
What do you think about a `finalize_dogmatic_config`?
In this way it is easier to switch between example configs.

Reply:
yes, I though about that but I did not want to use it in an example since it is a little bit hard to follow

Reply:
You can also make sure that encoder and decoder configs are synchronized in there